alidock/__init__.py

```diff
@@ -29,18 +29,19 @@ def __str__(self):
 
 class AliDock(object):
 
-    def __init__(self):
+    def __init__(self, overrideConf=None):
         self.cli = docker.from_env()
         self.lastUpdRelative = ".alidock_last_updated"
         self.dirInside = "/home/alidock"
         self.logRelative = ".alidock.log"
         self.conf = {
-            "dockName"     : "/alidock",
+            "dockName"     : "alidock",
```

Comment:
@mconcas the new Docker API is smart. We can drop the `/`, it's the same. We can get both `alidock` and `/alidock` and they will both work. If we ask for a _list_ however it will return `/alidock`.